framework/src/main/java/org/checkerframework/common/basetype/messages.properties

```diff
@@ -46,24 +46,24 @@ type.arguments.not.inferred=Could not infer type arguments for %s.
 unallowed.access=access of the field (%s) is not permitted on receiver of type (%s)
 cast.redundant=Redundant cast;\ntype   : %s
 
-purity.deterministic.void.method=a deterministic method without return value is redundant
-purity.deterministic.constructor=a deterministic constructor is redundant
-purity.invalid.overriding=Incompatible purity declaration\nMethod\n  %s in %s\n  cannot override\n  %s in %s\nfound   : %s\nrequired: %s
+# TODO: The call.{constructor,method} messages should take an argument indicating which method or constructor is the problem
```

Comment:
These errors are raised on the method invocation that is problematic. Repeating that seems redundant.

Reply:
I added this because the information in the current messages is not sufficient.  For example, it is helpful to inform the user override is being invoked.